components/Progress/Progress.tsx

```diff
@@ -48,7 +54,7 @@ const Progress: React.FC<ProgressProps> = ({
   return (
     <Box position="relative">
       <ChakraProgress
-        height={8}
+        height={barHeight ?? 8}
```

Comment:
if you take a look at line 64 (`...restProps`), you're already able to override the `height` prop by passing it to the component itself

e.g. `<Progress height={10} />` in the dapp will pass `10` through automatically here without the need for an extra prop for `barHeight` to be manually added

lmk if that doesn't work though and i can investigate further

Reply:
Oh yikes yeah just reverted the changes, thanks!